webdev/nextjs/webdev.sh

```diff
@@ -157,15 +138,15 @@ case "$1" in
         npx next build  || exit $?
         npx next export || exit $?
         cd azion/cells-site-template || exit $?
-        azion-framework-adapter build --config ../kv.json \
+        npx --yes azion-framework-adapter@0.2.0 build --config ../kv.json \
                              --static-site --assets-dir ../../out || exit $? ;;
 
     publish )
+        check_init || exit $?
         check_envvars || exit $?
-        check_azion_framework_adapter || exit $?
 
         cd azion/cells-site-template || exit $?
         # Publish only assets
-        azion-framework-adapter publish --config ../kv.json \
+        npx --yes azion-framework-adapter@0.2.0 publish --config ../kv.json \
```

Comment:
```suggestion
        npx --yes azion-framework-adapter@0.2.0 publish --config ../kv.json \
```
Dropping the patch number should allow automatic to update the npm package  for bugs.

Reply:
https://github.com/aziontech/azioncli-template/pull/35#discussion_r1001968121